src/Microsoft.Extensions.Http/DependencyInjection/HttpClientFactoryServiceCollectionExtensions.cs

```diff
@@ -183,9 +184,6 @@ public static IHttpClientBuilder AddHttpClient(this IServiceCollection services,
         /// can be retrieved from <see cref="IServiceProvider.GetService(Type)" /> (and related methods) by providing
         /// <typeparamref name="TClient"/> as the service type. 
         /// </para>
-        /// <para>
-        /// Use <see cref="Options.Options.DefaultName"/> as the name to configure the default client.
-        /// </para>
```

Comment:
This wasn't correct. This paragraph is only supposed to be present on the overloads that accept the name as a string.